id-repository/credential-request-generator/src/main/resources/bootstrap.properties

```diff
@@ -27,8 +27,8 @@ javax.persistence.jdbc.driver=${mosip.credential.service.jdbc.driver}
 #jakarta.persistence.jdbc.user=${mosip.credential.service.jdbc.user}
 #jakarta.persistence.jdbc.password=${mosip.credential.service.jdbc.password}
 #jakarta.persistence.jdbc.driver=${mosip.credential.service.jdbc.driver}
-spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
-spring.batch.job.enabled=false
+# spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect

```

Comment:
after removing this property, please test the service is working fine or not

Reply:
Yes tested locally not facing any issue